.github/workflows/core_tests.yml

```diff
@@ -30,4 +30,7 @@ jobs:
         run: |
           pytest test/test_skill_utils.py
         env:
-          GITHUB_TOKEN: ${{secrets.neon_token}}
\ No newline at end of file
+          GITHUB_TOKEN: ${{secrets.neon_token}}
+      - name: Test Language
```

Comment:
I would suggest generating xml artifacts on each test run to check if something goes wrong

Reply:
Opened a separate issue for this since it should apply to all tests